.circleci/config.yml

```diff
@@ -102,7 +113,7 @@ commands:
       - docker/build:
           step-name: << parameters.step-name >>
           image: << parameters.image-name >>
-          extra_build_args: "--build-arg NF_IMAGE_VERSION=${CIRCLE_SHA1} --build-arg NF_IMAGE_TAG=${BRANCH} --target << parameters.build-target >> <<# parameters.squash >> --squash <</ parameters.squash >>"
+          extra_build_args: "--build-arg NF_IMAGE_VERSION=${CIRCLE_SHA1} --build-arg NF_IMAGE_TAG=${VERSION_TAG} --target << parameters.build-target >> <<# parameters.squash >> --squash <</ parameters.squash >>"
```

Comment:
Make the `NF_IMAGE_TAG` more specific. Internally this is used by our Dockerfile and scripts as a version tag (which gets printed in logs etc.). For branch build this is going to be the branch name but for git tag builds this should be that git tag (see the changes within the `compute-docker-tags` commands.